openmdao/core/notebook_mode.py

```diff
@@ -0,0 +1,23 @@
+"""Checking for interactive notebook mode."""
+
+
+def _notebook_mode():
```

Comment:
Tried on my branch. I don't have tabulate installed. I get

`Tabulate is not installed run `pip install openmdao[notebooks]` to install required dependencies`

every time I run an openmdao model.